.github/workflows/build.yml

```diff
@@ -129,6 +140,7 @@ jobs:
     name: BrowserStack
     needs: job_build
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
BrowserStack is wicked slow - 15 min prob isn't enough here.

Reply:
Are those tests even used? It hasn't been passing for awhile it seems (Novemberish?)

Reply:
Yeah, I was looking at that earlier. It's a good question. I've made an Asana task to either fix or ditch it. In the meantime, I guess it's fine to leave it at 15 minute. It'll time out, but only if its individual tests haven't _already_ timed out, which they seem to do almost always. So, no real loss, I suppose.

Reply:
Ooops, I do think you're right and we should increase it, but was also curious about them in general.

Reply:
```suggestion
    timeout-minutes: 45
```